integration-tests/smoke/forwarder_ocr_test.go

```diff
@@ -117,7 +82,7 @@ var _ = Describe("OCR forwarder flow - each operator forwarder pair belongs to e
 		Expect(err).ShouldNot(HaveOccurred(), "Deploying Link Token Contract shouldn't fail")
 
 		By("Funding Chainlink nodes")
-		err = actions.FundChainlinkNodes(chainlinkNodes, chainClient, funding)
+		err = actions.FundChainlinkNodes(chainlinkNodes, chainClient, big.NewFloat(.05))
```

Comment:
We had different funding values depending on the network before. Is this magic number good for all networks now some how? I see similar things in the other test files.

Reply:
So, this is tricky either way. Best to discuss it elsewhere.